services/worker/src/worker/job_runners/dataset/dataset_job_runner.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         if job_info["params"]["revision"] is None:
             raise ParameterMissingError("'revision' parameter is required")
         self.dataset = job_info["params"]["dataset"]
-        self.revision = job_info["params"]["revision"]
+        self.dataset_git_revision = job_info["params"]["revision"]
```

Comment:
was it a bug? does it mean we should fix some cache entries?

Reply:
Yes, it was a bug. But I think it does not affect the cache entries since the field self.dataset_git_revision was not used anywhere (not in job runner nor in cache record).
I started using it in this PR for first-rows-from-streaming and first-rows-from-parquet job runners.